akka-docs/rst/java/persistence.rst

```diff
@@ -7,7 +7,7 @@ Persistence
 Akka persistence enables stateful actors to persist their internal state so that it can be recovered when an actor
 is started, restarted after a JVM crash or by a supervisor, or migrated in a cluster. The key concept behind Akka
 persistence is that only changes to an actor's internal state are persisted but never its current state directly
-(except for optional snapshots). These changes are only ever appended to storage, nothing is ever mutated, which
+(except for optional snapshots). These changes are only ever appended to storage, nothing is ever mutated which
```

Comment:
that comma actually belongs there, according to http://grammartips.homestead.com/nonrestrictivecommas.html